packages/@aws-cdk/aws-appmesh/lib/http-route-path-match.ts

```diff
@@ -0,0 +1,90 @@
+import { CfnRoute } from './appmesh.generated';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from '@aws-cdk/core';
+
+/**
+ * Path and Prefix properties for HTTP route match.
+ */
+export interface HttpRoutePathMatchConfig {
+  /**
+   * Route CFN configuration for path match.
+   *
+   * @default - no path match.
+   */
+  readonly pathMatch?: CfnRoute.HttpPathMatchProperty;
+
+  /**
+   * String defining the prefix match.
+   *
+   * @default - no prefix match
+   */
+  readonly prefixMatch?: string;
```

Comment:
If we're going with `pathMatch` above, this needs to be called `prefixPathMatch`.

Reply:
Addressed